### ISSUE TITLE
How to reproduce LSTM result?

### ISSUE THREAD

**Issue by felixfuu** [flennerhag/alstm#4]
Thank you very much for the code. But, how do I reproduce the results in the Table2?(LSTM,20M,2,71.7,68.9)

pytorch version is >=0.4.0

**Comment by flennerhag** [comment]
The code base changed slightly prior to public release, and the result was obtained with pytorch 0.2.0, but you should get identical performance by running

```bash
python main.py --model LSTM --epochs 200 --emsize 400 --nhid 1650 --nlayers 2 --dropouth 0.20 --dropoute 0.16 --dropouti 0.48 --dropouto 0.49 --wdecay 1e-6 --clip 0.2 --var-seq --seq-len 70 --batch_size 20 --cut-steps 100 140 155 165 175 --cut-rate 10 --save
```

You might have to reduce weight decay to ``1e-7``.
